packages/spec/src/spec.emu.html

```diff
@@ -154,14 +154,28 @@ <h1>Lexical Grammar</h1>
 StringLiteral :
     `"` StringCharacters? `"`
     `"""` TripleQuotedStringCharacters? `"""`
+    StringTemplateHead
+
+StringTemplateHead ::
```

Comment:
I think this grammar is incomplete because we cannot reach StringTemplateMiddle/StringTemplateTail from the top-level goal symbol InputElement. We will need a separate goal symbol (say, `InputElementTemplateTail`). I also think it should be a parse error to have an empty substitution, which would require separating out right brace from the list of tokens and only allowing it with the `InputElementTemplateTail`.

Reply:
updated, let me know if that looks better